File: URI/AD-HOC/2343 - Caçadores de Mitos.py
# Autor: Gustavo Policarpo<gustavo_votagus@hotmail.com>
# Nome: Caçadores de Mitos
# Nível: 5
# Categoria: AD-HOC
# URL: https://www.urionlinejudge.com.br/judge/pt/problems/view/2343

n = input()
mat = []
for i in range (501):
	mat.append([0]*501)
yep = 0
for i in range (n):
	x, y = map( int, raw_input().split() )
	if mat[x][y] == 1:
		yep=1
	mat[x][y]=1
if yep == 1:
	print 1
else:
	print 0

File: URI/AD-HOC/2343 - Caçadores de Mitos.cpp
// Author: Gustavo Policarpo
// Name: Caçadores de Mitos
// Level: 5
// Category: AD-HOC
// URL: https://www.beecrowd.com.br/judge/en/problems/view/2343

#define gc getchar
#define pc(x) putchar(x);
#include <bits/stdc++.h>

//LIFE IS NOT A PROBLEM TO BE SOLVED

using namespace std;

#define rep(i,a,b) for(int i = int(a); i < int(b) ; i++)
#define reo(i,a,b) for(int i = int(a); i > int(b) ; i--)
#define rei(i,a,b) for(int i = int(a); i < int(b) ; i=i)
#define pb push_back
#define mp make_pair
#define F first
#define S second

#define INF 0x3F3F3F3F
#define MAXV 100001
#define mod 1000000007

typedef long long int ll;
typedef unsigned long long int ull;
typedef pair < int, int > ii;
typedef pair < int, ii > iii;

void scanint(int &x)
{
    register int ccc = gc();
    x = 0;
    int neg = 0;
    for(;((ccc<48 || ccc>57) && ccc != '-');ccc = gc());
    if(ccc=='-') {neg=1;ccc=gc();}
    for(;ccc>47 && ccc<58;ccc = gc()) {x = (x<<1) + (x<<3) + ccc - 48;}
    if(neg) x=-x;
}



int mat[505][505];
int main()
{
	int N; scanint(N);
	int x, y, yep=0;
	while(N--){
		scanint(x), scanint(y);
		if(mat[x][y]) yep=1;
		mat[x][y]=1;
	}
	
	puts(yep? "1" : "0");
	
	return 0;
	
}
